python/cuml/cuml/tests/test_sklearn_import_export.py

```diff
@@ -81,6 +82,8 @@ def assert_estimator_roundtrip(
     # Convert back
     roundtrip_model = type(cuml_model).from_sklearn(sklearn_model)
 
+    assert original_params == roundtrip_model.get_params()
```

Comment:
I think this should be true. Otherwise calling `fit`, which is possible (or should we disallow it? A la `FrozenEstimator`), will lead to very weird results.